### PR TITLE
Getting Started Improvements

### PR DESCRIPTION
**Pull request by NirantK**
1. Rewrite to use TextEmbedding instead of DefaultEmbeddine
2. Align the README and Getting Started snippets

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,13 @@ FastEmbed is a lightweight, fast, Python library built for embedding generation.
...
-    - ONNX Runtime, no PyTorch dependency
-    - CPU-first design
-    - Data-parallelism for encoding of large datasets
+## 📈 Why FastEmbed?
 
-2. Accuracy/Recall
-    - Better than OpenAI Ada-002
-    - Default is Flag Embedding, which is top of the [MTEB](https://huggingface.co/spaces/mteb/leaderboard) leaderboard
-    - List of [supported models](https://qdrant.github.io/fastembed/examples/Supported_Models/) - including multilingual models
+1. Light: FastEmbed is a lightweight library with few external dependencies. Including on serverless runtimes like AWS Lambda. We don't require a GPU and don't download GBs of PyTorch dependencies, and instead use the ONNX Runtime.
```
Comment (Anush008) [inline README.md:9]:
```suggestion
Including on serverless runtimes like AWS Lambda.
```
This is off I think.

Comment (NirantK) [reply README.md:9]: Why? FastEmbed has a low footprint making it great for serverless runtimes — have spoken to users who use it exclusively in AWS Lambda, that's where I got the confidence to put it here.

Comment (NirantK) [reply README.md:9]: Or did you mean the sentence/grammar?

File: docs/Getting Started.ipynb [hunk 2]
```diff
@@ -39,186 +33,186 @@
...
-  {
-   "cell_type": "markdown",
-   "id": "1cb3cc87",
-   "metadata": {},
-   "source": [
-    "### Load the Embedding Model Weights\n",
-    "Next, initialize the Embedding class with the desired parameters. Here, \"BAAI/bge-small-en\" is the pre-trained model name, and max_length=512 is the maximum token length for each document.\n",
-    "\n",
-    "This will download the model weights, decompress to directory `local_cache` and load them into the Embedding class.\n",
+    "- **Passages**: Add \"passage:\" at the beginning of each passage string\n",
```
Comment (Anush008) [inline docs/Getting Started.ipynb:157]: Not the case every time.

Comment (NirantK) [reply docs/Getting Started.ipynb]: Added the note that this is for the default model

**Review by Anush008 — APPROVED** [review]

**Pull request merged**